src/util/CompareArrays.js

```diff
@@ -0,0 +1,10 @@
+//a function to compare two arrays with objects ->
+//array1 (userIntolerances) containing objects that should be deleted from array2 (allPossibleIntolerances)
```

Comment:
I really like that you want to explain what the function is doing. Try this `best practise` format instead:

```js
/**
 * Returns a filtered array.
 *
 * @param {array} userIntolerances An array with user intorlerances.
 * @param {array} allPossibleIntolerances Another array with infos in it.
 * @return {array}  a filtered array.
 */

...code
```

You have to put in real information, I was just tinkering around.

Reply:
aaaaaaaaaaaaaaaaaaaaand good solution. Is it working? can you write tests and check some edge cases?

Reply:
ok thanks :) 
I tested it manually and it works - but I'll add a test too 👍